packages/portal-proto/src/components/DownloadButtons/DownloadFile.tsx

```diff
@@ -59,6 +61,7 @@ export const DownloadFile: React.FC<DownloadFileProps> = ({
   if (file.access === "open") {
     return (
       <DownloadButton
+        data-testid={customDataTestID}
```

Comment:
Is this part working? Are you able to see the data-testid?

Reply:
Yes it appears for open/controlled access file summaries and for the little download file buttons

Reply:
@bpierce01 can you give me an example on how this transforms to for the element?

Reply:
I feel like we would have to set a prefix, such as `button-${customDataTestID}`. Please correct me if I am wrong.